packages/server/src/api/util/catch-async-route.ts

```diff
@@ -13,8 +16,7 @@ export default function catchAsyncRoute(
             const result = await controllerFn(req, res);
             return res.json(result);
         } catch (err) {
-            console.log('Error in Route:', err);
-            res.status(err.status).send(err);
+            next(err);
```

Comment:
@kkennis looks like this change made it back in some how and i had to revert back to using `next(err)`